### ISSUE TITLE
Could content_path be set per-controller, instead of needing a custom page finder?

### ISSUE THREAD

**Issue by nruth** [thoughtbot/high_voltage#217]
Currently the content path is a global variable in the configuration file. This is ok if you only use high-voltage for one route.

If you use it in two places I think something like this is needed:

```
class Admin::StyleguideController < ApplicationController
  before_filter :authenticate_admin!
  include HighVoltage::StaticPage


  private

  def page_finder_factory
    Admin::StyleguidePageFinder
  end
end

class Admin::StyleguidePageFinder < HighVoltage::PageFinder
  def content_path
    '/admin/styleguide/'
  end
end
```

But the obvious thing would be to do something like this:

```
class Admin::StyleguideController < ApplicationController
  before_filter :authenticate_admin!
  include HighVoltage::StaticPage
  def content_path
    '/admin/styleguide/'
  end
end
```

Is this something that's not come up, or that was decided against to keep things modular?

I'd like to either:
- PR the readme showing how to set per-controller with a custom finder
- PR where the content_path can be overridden per-controller

**Comment by dgalarza** [comment]
I believe what your looking for in the read me is the section on [Custom Finding](https://github.com/thoughtbot/high_voltage#custom-finding) which explains how to set a custom finder per-controller.

**Comment by nruth** [comment]
Hi, thanks for looking. 

Yes that part of the readme does give the desired result, it's how I came up with the first code block above. 

However, I don't think it's straightforward for beginners, as there are some missing steps and it doesn't explicitly mention solving the mount-2-paths problem.

The API verbosity comment is more of a RFC. I'd like to learn why it was built like that, or see if the shortcut I suggested is a bad idea or would be an interesting PR.

I'm fine with this staying closed though, as it isn't a bug.
